sdk/core/azure-core-test/src/main/java/com/azure/core/test/http/PlaybackClient.java

```diff
@@ -165,4 +171,15 @@ private static String removeHost(String url) {
 
         return String.format("%s%s", urlBuilder.path(), urlBuilder.queryString());
     }
+
+    private void constructExceptionType(final NetworkCallRecord networkCallRecord) throws Exception {
+        if (networkCallRecord.exception() == null) {
+            return;
+        }
+        if (networkCallRecord.exception() != null && networkCallRecord.exception().argTypes() == null) {
```

Comment:
We don't need the not null check, already checked for being null right before this

Reply:
It is checking networkCallRecord being null or not. There is no checking for networkCallRecord.exception()

Reply:
Right but you check for null and return them check for not null which will always be true.

Reply:
Right. I am reading wrong line.
Fixed it.